### DISCUSSION TITLE
The autobackup property can have 3 values - child option?

### DISCUSSION THREAD

**Discussion by ozgurerdogan** [psy0rz/zfs_autobackup#162] [General]
Dataset contains children dataset only? What else? So what is this option for?
child: Only backup the children off the dataset, not the dataset itself.

Is it to exclude if dataset has any mount points?

**Comment by psy0rz** [comment]
That was a typo in the documentation, should read 'only backup the childeren of the dataset, not the dataset itself'

So if you:
`
zfs set autobackup:test=child somepool/test
`

it will only backup somepool/test/* and not somepool/test

if you
`
zfs set autobackup:test=true somepool/test
`

then it will backup somepool/test as well as somepool/*

**Comment by ozgurerdogan** [comment]
Ok. Sometime I may need to create mountpoints like:
zfs create rpool/data/backups -o mountpoint=/P1BACKUP
To store iso files. So is this setting has anything to backup these iso file when using "true" ?
